base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleDataResponse.java

```diff
@@ -372,22 +373,40 @@ public void setCloudAppVehicleID(String cloudAppVehicleID){
     public String getCloudAppVehicleID(){
         return getString(KEY_CLOUD_APP_VEHICLE_ID);
     }
-
-
-    /**
-     * Sets a value for OEM Custom VehicleData.
-     * @param vehicleDataName a String value
-     * @param vehicleDataState a VehicleDataResult value
-     */
-    public void setOEMCustomVehicleData(String vehicleDataName, Object vehicleDataState){
-        setParameters(vehicleDataName, vehicleDataState);
-    }
-
-    /**
-     * Gets a VehicleData value for the vehicle data item.
-     * @return a Object related to the vehicle data
-     */
-    public Object getOEMCustomVehicleData(String vehicleDataName){
-        return getParameters(vehicleDataName);
-    }
-}
+
+
+    /**
+     * Sets a value for OEM Custom VehicleData.
+     * @param vehicleDataName a String value
+     * @param vehicleDataState a VehicleDataResult value
+     */
+    public void setOEMCustomVehicleData(String vehicleDataName, Object vehicleDataState){
+        setParameters(vehicleDataName, vehicleDataState);
+    }
+
+    /**
+     * Gets a VehicleData value for the vehicle data item.
+     * @return a Object related to the vehicle data
+     */
+    public Object getOEMCustomVehicleData(String vehicleDataName){
+        return getParameters(vehicleDataName);
+    }
+
+    /**
+     * Sets an array of statuses for WindowStatus.
+     * @param status a WindowStatus value
+     */
+    public void setWindowStatus(@NonNull List<WindowStatus> status){
+        setParameters(KEY_WINDOW_STATUS, status);
+    }
+
+    /**
+     * Gets a List<WindowStatus> value for WindowStatus.
+     * @return a list of WindowStatus object or null.
+     * If true, means the WindowStatus data has been subscribed.
+     */

```

Comment:
```suggestion
/**
     * Gets the windowStatus. If true, means the WindowStatus data has been subscribed.
     *
     * @return List<WindowStatus> See WindowStatus
     * @since SmartDeviceLink 7.0.0
     */
```
Please use the [RPC generator](https://github.com/smartdevicelink/sdl_java_suite/tree/master/generator) in the sdl_java_suite project to ensure the RPC matched the spec exactly.